README.md

```diff
@@ -3,62 +3,188 @@ page_type: sample
+    - Registered application's **Client secret**
+1. Build the application with `dotnet build`:
+
+    ```console
+    azureuser@machine:~/ms-identity-dotnetcore-b2c-account-management$ cd src
+    azureuser@machine:~/ms-identity-dotnetcore-b2c-account-management/src$ dotnet build
+    Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
+    Copyright (C) Microsoft Corporation. All rights reserved.
+
+      Restore completed in 431.62 ms for /home/azureuser/ms-identity-dotnetcore-b2c-account-management/src/b2c-ms-graph.csproj.
+      b2c-ms-graph -> /home/azureuser/ms-identity-dotnetcore-b2c-account-management/src/bin/Debug/netcoreapp3.0/b2c-ms-graph.dll
+
+    Build succeeded.
+        0 Warning(s)
+        0 Error(s)
+
+    Time Elapsed 00:00:02.62
+    ```
 
 ## Running the sample
 
-Outline step-by-step instructions to execute the sample and see its output. Include steps for executing the sample from the IDE, starting specific services in the Azure portal or anything related to the overall launch of the code.
+Execute the sample with `dotnet b2c-ms-graph.dll`, select the operation you'd like to perform, then press ENTER.
+
+For example, get a user by object ID (command `2`), then exit the application with `exit`:
+
+```console
+azureuser@machine:~/ms-identity-dotnetcore-b2c-account-management/src$ dotnet bin/Debug/netcoreapp3.0/b2c-ms-graph.dll
+
+Command  Description
+====================
+[1]      Get all users (one page)
+[2]      Get user by object ID
+[3]      Get user by sign-in name
+[4]      Delete user by object ID
+[5]      Update user password
+[6]      Create users (bulk import)
+[help]   Show available commands
+[exit]   Exit the program
+-------------------------
+Enter command, then press ENTER: 2
+Enter user object ID: 064deeb8-0000-0000-0000-bf4084c9325b
+Looking for user with object ID '064deeb8-0000-0000-0000-bf4084c9325b'...
+{"displayName":"Autumn Hutchinson","identities":[{"signInType":"emailAddress","issuer":"contosob2c.onmicrosoft.com","issuerAssignedId":"autumn@fabrikam.com","@odata.type":"microsoft.graph.objectIdentity"},{"signInType":"userPrincipalName","issuer":"contosob2c.onmicrosoft.com","issuerAssignedId":"064deeb8-0000-0000-0000-bf4084c9325b@contosob2c.onmicrosoft.com","@odata.type":"microsoft.graph.objectIdentity"}],"id":"064deeb8-0000-0000-0000-bf4084c9325b","@odata.type":"microsoft.graph.user","@odata.context":"https://graph.microsoft.com/beta/$metadata#users(displayName,id,identities)/$entity","responseHeaders":{"Date":["Fri, 14 Feb 2020 18:52:56 GMT"],"Cache-Control":["no-cache"],"Transfer-Encoding":["chunked"],"Strict-Transport-Security":["max-age=31536000"],"request-id":["23165c3f-0000-0000-0000-f7fc59669c24"],"client-request-id":["23165c3f-0000-0000-0000-f7fc59669c24"],"x-ms-ags-diagnostic":["{\"ServerInfo\":{\"DataCenter\":\"WEST US 2\",\"Slice\":\"E\",\"Ring\":\"1\",\"ScaleUnit\":\"000\",\"RoleInstance\":\"MW1PEPF00001671\"}}"],"OData-Version":["4.0"]},"statusCode":"OK"}

```

Comment:
>issuer [](start = 79, length = 6)

nit: where do you get the issuer value? in the token is looks like this: `[https://fabrikamb2c.b2clogin.com/775527ff-9a37-4307-8b3d-cc311f58d925/v2.0`](https://fabrikamb2c.b2clogin.com/775527ff-9a37-4307-8b3d-cc311f58d925/v2.0`) should they be more similar?

Reply:
@jennyf19 That's the deserialized output directly from Graph:

```csharp
// Get user by object ID
var result = await graphClient.Users[userId]
    .Request()
    .Select(e => new
    {
        e.DisplayName,
        e.Id,
        e.Identities
    })
    .GetAsync();
    
if (result != null)
{
    Console.WriteLine(JsonConvert.SerializeObject(result));
}
```


https://docs.microsoft.com/graph/api/user-list?view=graph-rest-1.0&tabs=csharp#example-2-users-request-using-select

Reply:
@mmacy ok...that makes sense then. thanks for clarifying.